Title: 106620 – pyton wxpython 2.6.0.0-r1 import error
URL: https://bugs.gentoo.org/show_bug.cgi?id=106620

Go to: 
		Gentoo Home
Documentation
Forums
Lists
Bugs
Planet
Store
Wiki
Get Gentoo!





Gentoo's Bugzilla – Bug 106620
pyton wxpython 2.6.0.0-r1 import error
Last modified: 2006-09-24 06:56:05 UTC node [vulture]


Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]













Bug 106620 
      - pyton wxpython 2.6.0.0-r1 import error


Summary:
pyton wxpython 2.6.0.0-r1 import error
    








Status:
    

RESOLVED
          WORKSFORME
      






Alias:


        None
    





Product:

Gentoo Linux




Classification:

Unclassified




Component:

[OLD] Library

  (show other bugs)



Hardware:

x86
        Linux
      







Importance:
      
High
       normal
      


Assignee:

Gentoo wxWidgets project








URL:







Whiteboard:




Keywords:








Depends on:







Blocks:









 





      Reported:
    
2005-09-19 15:24 UTC by Yoandy Rodriguez Martinez





      Modified:
    
2006-09-24 06:56 UTC
      (History)
    




          CC List:
        

2 
          users
          
            (show)
          



kayos
python









See Also:






Package list:







Runtime testing required:

---























      Attachments
    



Add an attachment
        (proposed patch, testcase, etc.)
    








Note
              You need to
              log in
              before you can comment on or make changes to this bug.
            

















Description


Yoandy Rodriguez Martinez





          2005-09-19 15:24:23 UTC
        

When importing wx or wxPython using python 2.3.5 and wxpython 2.6.0.0-r1 with
wxGTK 2.6.1 a ImportError exception gets launched.

Reproducible: Always
Steps to Reproduce:
1.python -c "import wx"
2.
3.

Actual Results:  
Traceback (most recent call last):
  File "<stdin>", line 1, in ?
  File "/usr/lib/python2.3/site-packages/wx-2.6-gtk2-unicode/wx/__init__.py",
line 42, in ?
    from wx._core import *
  File "/usr/lib/python2.3/site-packages/wx-2.6-gtk2-unicode/wx/_core.py", line
4, in ?
    import _core_
ImportError: /usr/lib/python2.3/site-packages/wx-2.6-gtk2-unicode/wx/_core_.so:
undefined symbol: wxEVT_SCROLL_ENDSCROLL



Portage 2.0.51.22-r2 (default-linux/x86/2005.0, gcc-3.4.4, glibc-2.3.5-r0,
2.6.10-gentoo-r7 i686)
=================================================================
System uname: 2.6.10-gentoo-r7 i686 Intel(R) Pentium(R) 4 CPU 2.40GHz
Gentoo Base System version 1.4.16
distcc 2.16 i686-pc-linux-gnu (protocols 1 and 2) (default port 3632)
[disabled]ccache version 2.3 [enabled]
dev-lang/python:     2.3.5
sys-apps/sandbox:    1.2.12
sys-devel/autoconf:  2.13, 2.59-r6
sys-devel/automake:  1.4_p6, 1.5, 1.6.3, 1.7.9-r1, 1.8.5-r3, 1.9.5
sys-devel/binutils:  2.16.90.0.3
sys-devel/libtool:   1.5.16
virtual/os-headers:  2.6.8.1-r2
ACCEPT_KEYWORDS="x86"
AUTOCLEAN="yes"
CBUILD="i686-pc-linux-gnu"
CFLAGS="-march=i686 -O2 -pipe"
CHOST="i686-pc-linux-gnu"
CONFIG_PROTECT="/etc /usr/kde/2/share/config /usr/kde/3.3/env
/usr/kde/3.3/share/config /usr/kde/3.3/shutdown /usr/kde/3/share/config
/usr/lib/X11/xkb /usr/lib/mozilla/defaults/pref /usr/share/config
/var/qmail/control"
CONFIG_PROTECT_MASK="/etc/gconf /etc/terminfo /etc/env.d"
CXXFLAGS="-march=i686 -O2 -pipe"
DISTDIR="/usr/portage/distfiles"
FEATURES="autoaddcvs autoconfig ccache distlocks notitles sandbox sfperms strict
userpriv usersandbox"
GENTOO_MIRRORS="ftp://10.33.20.155/portage"
MAKEOPTS="-j2"
PKGDIR="/usr/portage/packages"
PORTAGE_TMPDIR="/var/tmp"
PORTDIR="/usr/portage"
PORTDIR_OVERLAY="/usr/local/portage"
SYNC="rsync://10.7.7.155/nova-portage"
USE="x86 X acpi alsa apm avi bash-completion berkdb bitmap-fonts cdr crypt cups
curl directfb emboss encode esd fam firefox flac foomaticdb fortran gcj gdbm gif
gnome gpm gstreamer gtk gtk2 hal imagemagick imlib ipv6 java jikes jpeg junit
ldap libg++ libwww mad mikmod mmx mono motif mp3 mpeg nas ncurses nls nptl ogg
oggvorbis opengl oss pam pdflib perl png postgres python quicktime readline
samba sdl slang spell sqlite sse sse2 ssl svga tcltk tcpd tiff truetype
truetype-fonts type1-fonts vorbis win32codecs xml xml2 xmms xv zlib userland_GNU
kernel_linux elibc_glibc"
Unset:  ASFLAGS, CTARGET, LANG, LC_ALL, LDFLAGS, LINGUAS




Comment 1


Rob Cakebread (RETIRED)






          2005-10-11 15:56:49 UTC
        

What version of x11-libs/gtk+ are you using? 

You might want to try wxpython-2.6.1.0 as it contains quite a few bug fixes
(should be marked x86 stable this week) 




Comment 2


Mart Raudsepp






          2006-09-24 06:53:47 UTC
        

Reassigning to wxwindows, as that herd has become the primary maintainer of wxpython now.




Comment 3


Mart Raudsepp






          2006-09-24 06:56:05 UTC
        

This works for me, and I indeed recall some issues in EVT_SCROLL stuff back in the 2.6.0 days.
2.6.1 is stable and should be free of this problem, as should be newer versions.

Closing as WORKSFORME in the lack of an OUTDATED option.
Please reopen if it is still broken.









Format For Printing
 - XML
 - Clone This Bug
 - Clone In The Same 
                        Product
 - Top of page 







Home
| New–[Ex]
| Browse
| Search
| Privacy Policy

| 





[?]
| Reports

| 
Requests

| 
Help


| 
New Account


| 
Log In





[x]



| 
Forgot Password

Login:




[x]

